kura/distrib/pom.xml

```diff
@@ -1950,5 +1961,72 @@
             </build>
         </profile>
 
+        <profile>
+            <properties>
+                <kura.skip.if.target.definition.profile>true</kura.skip.if.target.definition.profile>
```

Comment:
why we need to skip this and not include in the build all?

I would also expect the artifacts produced to be in a target folder, isn't it?